coralproject/talk
# Facebook Auth Plugin

## What does this PR do?

- Extracts server side facebook auth into plugin, enabled by default

## How do I test this PR?

- Try with facebook auth plugin enabled
- Notice it works :)
- Try without facebook auth plugin enabled
- Notice that facebook auth does not work :)
